events/message.js

```diff
@@ -3,6 +3,24 @@ const { prefix, colors } = require('../config.json')
 
 const reactMessage = require('../utils/reactMessage')
 
+cmdQueue = []
+handlingCommand = false
+
+var waitToRunCommand = function() {
+  setTimeout(runCommand, 300) 
+}
+
+var runCommand = function() {
+  handlingCommand = true
```

Comment:
Could this not be handled by Promises / Observables / Callbacks to prevent having to manage a flag?

Reply:
It probably could, I'm not super familiar with how to do that. You're welcome to make changes if you'd like. Or you could try to walk me through that

Reply:
How can I add to this PR?

Reply:
It seems like you have to add me as a collaborator to your forked repo, that way I can push to your PR branch.